Sources/Hyperwallet.swift

```diff
@@ -630,7 +630,7 @@ public final class Hyperwallet: NSObject {
     /// - Parameters:
     ///   - queryParam: the ordering and filtering criteria
     ///   - completion: the callback handler of responses from the Hyperwallet platform
-    public func listPrepaidCards(queryParam: HyperwalletPrepaidCardQueryParam? = nil,
+    public func listPrepaidCards(queryParam: HyperwalletPrepaidCardQueryParm? = nil,
```

Comment:
This is a typo and has been fixed, why are you trying to again revert the fixed change?

Reply:
I got merge conflict . So I make file name same as development branch.

Reply:
Type fixed